sickbeard/tv.py

```diff
@@ -705,7 +705,7 @@ def load_episodes_from_db(self):
                     cur_ep.delete_episode()
 
                 cur_ep.load_from_db(cur_season, cur_episode)
-                cur_ep.load_from_indexer(tvapi=t, cached_season=cached_seasons[cur_season])
+                #cur_ep.load_from_indexer(tvapi=t, cached_season=cached_seasons[cur_season])
```

Comment:
i had to remove this too to avoid duplication

Can't understand why it loads from DB and then load from indexer